.github/workflows/DEPLOY_PLAYGROUND_PREVIEW.yml

```diff
@@ -0,0 +1,49 @@
+name: DEPLOY_PLAYGROUND_PREVIEW
+on:
+  pull_request:
+    types: [ labeled, synchronize ]
+jobs:
+  deploy-preview:
+    # check whether the labeled event was deploy-preview || check whether on new commit of PR the label deploy-preview exists
+    if: github.event.label.name == 'deploy-preview' || contains( github.event.pull_request.labels.*.name, 'deploy-preview')
+    runs-on: ubuntu-20.04
+    env:
+      BRANCH_NAME: ${{ github.head_ref || github.ref_name }} # head_ref = branch on PR, ref_name if master / stable/**
+      PREVIEW_BRANCH_NAME: demo-${{ github.head_ref || github.ref_name }}
+    steps:
+      - name: Check form-playground branch
+        run: echo "BRANCH_EXISTS=$(git ls-remote --heads https://${{ secrets.ADD_TO_HTO_PROJECT_PAT }}@github.com./camunda/form-playground ${{ env.PREVIEW_BRANCH_NAME }} | wc -l)" >> $GITHUB_ENV
+      - name: Create form-playground branch
+        if: env.BRANCH_EXISTS == 0
+        uses: GuillaumeFalourd/create-other-repo-branch-action@706963eca4b230707b1cfecee6760b519e2cbbf3
+        with:
+          repository_owner: camunda
+          repository_name: form-playground
+          new_branch_name: ${{ env.PREVIEW_BRANCH_NAME }}
+          access_token: ${{ secrets.ADD_TO_HTO_PROJECT_PAT }}
+      - name: Create GitHub deployment
+        uses: bobheadxi/deployments@88ce5600046c82542f8246ac287d0a53c461bca3
+        id: deployment
+        with:
+          step: start
+          env: ${{ env.PREVIEW_BRANCH_NAME }}
+          token: "${{ secrets.ADD_TO_HTO_PROJECT_PAT }}"
```

Comment:
you don't need to pass in a token.
GitHub will automatically supply you with a token on workflow start.
This token has some basic permissions in regards to your repository.

See [docs](https://docs.github.com/en/actions/security-guides/automatic-token-authentication) for more info.

tl;dr
Can be replaced with: ```${{ secrets.GITHUB_TOKEN }}```